### PR TITLE
Add changelog file

### LINKED ISSUE #201: Create a zos-lib changelog

### PR DESCRIPTION
**Pull request by facuspagnuolo**
Fixes #201 

Adding a changelog file following [keepachangelog guidelines](https://keepachangelog.com/en/1.0.0/)
Once merged I'll update GH release notes

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Improve test coverage
+- Improve arguments logging for proxies initialization
+- Replace colors dependency by chalk 
+
+#### Fixed
+- Make FileSystem copy function synchronous
+- Move solidity-coverage defaults params to Contracts object
+- Mark ethereumjs-abi as production dependency
+- Marl web3 as production dependency
+- Marl web3 as production dependency
```
Comment (spalladino) [inline changelog.md]: Marl => Mark

Comment (spalladino) [reply changelog.md]: Also duplicated

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### [1.3.0] - [date]
+
+#### Added
+- Support custom Truffle build directories
+- Reason messages to require statements
+- Proxy wrapper object to allow querying proxie's admin and implementation
```
Comment (spalladino) [inline changelog.md]: proxies'

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- Update OpenZeppelin to version 1.10.0
+- Update pragma solidity to version ^0.4.24
+- Use new solc constructor syntax
+
+#### Fixed
+- Fix migratable initializers ensuring they can never be run more than once
+- Fix freezable directory unset implementation method
+- Fix AdminUpgradeabilityProxy shadowing issues
+
+### [2.0.0] - [date]
```
Comment (frangio) [inline changelog.md]: Shouldn't this be in the `develop` branch instead of here?

File: changelog.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- Improve arguments logging for proxies initialization
+- Replace colors dependency by chalk 
+
+#### Fixed
+- Make FileSystem copy function synchronous
+- Move solidity-coverage defaults params to Contracts object
+- Mark ethereumjs-abi as production dependency
+- Marl web3 as production dependency
+- Marl web3 as production dependency
+- UpgradeabilityProxyFactory contract in-line documentation
```
Comment (frangio) [inline changelog.md]: Perhaps add nice formatting of identifiers? :slightly_smiling_face:

File: changelog.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### [1.3.0] - [date]
```
Comment (frangio) [inline changelog.md]: We don't need to add a version or `[date]` here. In the Keep a Changelog site they just list all changes under `[Unreleased]` as if it was a version number.

**Comment by facuspagnuolo** [comment]
@spalladino would you mind reviewing it again?

**Pull request merged**